projects/angular/src/data/datagrid/API.md

```diff
@@ -0,0 +1,53 @@
+| --clr-datagrid-popovers-box-shadow-color              | Changes the datagrid popovers shadow color.                                  |
+| --clr-datagrid-column-switch-header-font-color        | Changes the datagrid manage columns dialog header font color.                |
+| --clr-datagrid-column-switch-header-font-hover-color  | Changes the datagrid manage columns dialog header font color when hovered.   |
+| --clr-datagrid-detail-caret-icon-open-bg-color        | Changes the datagrid open detail caret icon background color.                |
+| --clr-datagrid-detail-caret-icon-open-icon-color      | Changes the datagrid open detail caret icon color.                           |
+| --clr-datagrid-placeholder-color                      | Changes the text color of datagrid placeholder.                              |
+| --clr-datagrid-placeholder-font-size                  | Changes the placeholder text size of datagrid.                               |
+| --clr-datagrid-pagination-btn-disabled-opacity        | Changes the opacity of pagination button when disabled.                      |
+| --clr-datagrid-column-toggle-border-color             | Changes the column toggle button border color.                               |
+| --clr-datagrid-column-toggle-fill-color               | Changes the background color of column toggle button.                        |
+| --clr-datagrid-column-toggle-text-color               | Changes the text color of column toggle button.                              |
+| --clr-datagrid-column-toggle-border-hover-color       | Changes the border color of toggle button when hovered over.                 |
+| --clr-datagrid-column-toggle-fill-hover-color         | Changes the background color of column toggle button when hovered over.      |
+| --clr-datagrid-column-toggle-text-hover-color         | Changes the text color of toggle button when hovered over.                   |
+| --clr-datagrid-column-toggle-border-active-color      | Changes the border color of toggle button when active.                       |
+| --clr-datagrid-column-toggle-fill-active-color        | Changes the background color of toggle button when active.                   |
+| --clr-datagrid-column-toggle-text-active-color        | Changes the text color of pagination toggle button when active.              |
+
+
+## Class names
+
+| Class name                            | Description                                                                     |
+| ------------------------------------- | -----------------------------------------------------------------------------   |
+| .clr-datagrid                         | Identify the main datagrid element.                                             |
+| .datagrid-nav                         | To filter and navigate through data displayed in datagrid.                      |
```

Comment:
This is demo-app only class, please remove. It's not part of the library.

Reply:
Removed as per your suggestion.